plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -105,10 +102,10 @@
 
     <!-- A "Connector" represents an endpoint by which requests are received
          and responses are returned. Documentation at :
-         Java HTTP Connector: /docs/config/http.html (blocking & non-blocking)
+         Java HTTP Connector: /docs/config/http.html
          Java AJP  Connector: /docs/config/ajp.html
          APR (HTTP/AJP) Connector: /docs/apr.html
-         Define a non-SSL HTTP/1.1 Connector on port 8080
+         Define a non-SSL/TLS HTTP/1.1 Connector on port 8080
     -->
     <Connector address="0.0.0.0" port="8080" protocol="org.apache.coyote.http11.Http11NioProtocol"
```

Comment:
hmm, not sur I'm convinced. Not adding it by default, because in some architectures, it may not be interesting...
If it's a predefined Tomcat feature, this will not harm us. In contrary, we should work on making our development ready to use it. We should upgrade to Servlet 4 for this and use HAProxy if needed (supports HTTP as reverse proxy).
So when the customers/community will request it, they will have it. This could be the very first step ;-)